src/registrar/utility/csv_export.py

```diff
@@ -202,7 +205,7 @@ def export_data_type_to_csv(csv_file):
         "State",
         "AO",
         "AO email",
-        "Security contact email",
+        # "Security contact email",
```

Comment:
I'M SORRY ITS ME AGAIN I think for testing purposes you may have commented it out, but I think we want to keep the security contact email for this data collection (I might be missing context so lmk if I'm incorrect) 😅

Reply:
**must change before merging** I agree looking at staging the security email is supposed to be here. Add it back in before merging